docs/azure-saas-docs/content/en/components/Identity/identity-framework-setup-manual.md

```diff
@@ -0,0 +1,410 @@
+---
+type: docs
+title: "Identity Framework Setup - Manual"
+weight: 5000
+toc_hide: true
+---
+
+On this page, you will find instructions for how to manually setup the Identity Framework through a combination of the Azure dashboard as well as relevant PowerShell commands
+
+# Setup Identity Framework - Manual (Advanced)
+
+## 1. Requirements
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/)
+- [Open SSL](https://www.openssl.org/)
+  - Note: An OpenSSL executable is also bundled with Git for Windows at `C:\Program Files\Git\usr\bin\openssl.exe`
+- [Visual Studio Code](https://code.visualstudio.com/)
+   - Required Extensions:
+   - [Azure AD B2C Custom Policy Extension](https://marketplace.visualstudio.com/items?itemName=AzureADB2CTools.aadb2c)
+
+## 2. Instructions
+
+### Step 1. Create B2C Tenant in Azure
+
+![(Azure Add Service Screenshot)](/azure-saas/images/identity-framework-manual-step-1-adding-service.png)
+
+- From the dashboard, click `Create a Resource` and select `Azure Active Directory B2C`
+- Note: A reference is registered in your subscription at time of creation to associate the tenant with your subscription.
+
+### Step 2. Log into newly created Tenant
+
+![(Switch Directory Screenshot)](/azure-saas/images/identity-framework-manual-step-2-switch-tenant-quicklink.png)
+
+- Click on `Open B2C Tenant` quicklink to immediately change directory to your new Tenant
+
+### Step 3. (Optional) Invite collaborators to the new Tenant
+
+Note: Collaborators are other developers you wish to help manage your services, not customers
+
+![(Tenant User Management Screenshot)](/azure-saas/images/identity-framework-manual-step-3-invite-collaborators.png)
+
+- Go to the `Azure AD B2C` dashboard while logged into your new Tenant, click `Users` in the lefthand navbar
+- After navigating, you may create or invite guest users into your Tenant allowing them access
+
+### Step 4. Create necessary app registrations in Azure AD B2C Tenant
+
+![(Registered Apps Screenshot)](/azure-saas/images/identity-framework-manual-step-4-app-registration.png)
+
+- From the Tenant `Azure AD B2C` dashboard, click `App registrations`
+- Add new registrations corresponding to the following list of modules
+- Instructions for adding client secrets, API scopes, and API permissions are detailed below
+
```

Comment:
Add a note here letting them know that they need to capture/copy down all IDs and secrets generated as part of this process for use later on in the setup.